egs/wsj/s5/steps/libs/nnet3/xconfig/basic_layers.py

```diff
@@ -343,7 +343,7 @@ def get_full_config(self):
         # unlike other layers the input layers need to be printed in
         # 'init.config' (which initializes the neural network prior to the LDA)
         ans = []
-        for config_name in [ 'init', 'ref', 'final' ]:
+        for config_name in [ 'ref', 'final' ]:
```

Comment:
here, we don't know if LDA is used or not. We don't add input info into 'init.config' immediately, but depend on judgement by outter info.